src/store/modules/account/actions.ts

```diff
@@ -15,6 +15,7 @@ import { getEURSPriceInWETH } from '@/services/chain/token-prices/token-prices';
 import { BuildExplorer } from '@/services/explorer';
 import { NetworkFeatureNotSupportedError } from '@/services/v2';
 import { CoinGeckoAPIService } from '@/services/v2/api/coinGecko';
+import { MoverAssetsService } from '@/services/v2/api/mover/assets/MoverAssetsService';
```

Comment:
```suggestion
import { MoverAssetsService } from '@/services/v2/api/mover/assets';
```
IMHO will be consistent with other services names